opentelemetry-api/tests/propagators/test_global_httptextformat.py

```diff
@@ -12,11 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import typing
 import unittest
 
 from opentelemetry import baggage, trace
-from opentelemetry.propagators import extract, inject
+from opentelemetry.propagators.util import extract, inject
```

Comment:
Will this work? I don't see `util` under propagators.

Reply:
Weird, fixing that.